pkg/operator/controllers/workaround/const.go

```diff
@@ -16,6 +16,10 @@ const (
 	workerMachineConfigPoolName = "worker"
 	memReserved                 = "2000Mi"
 
+	// 	machineconfiguration annotation added on the kubelet CRD - aro-limits
+	mcAnnotationName  = "machineconfiguration.openshift.io/mc-name-suffix"
```

Comment:
These are already declared in nodeautoresize in this PR. This should be moved to consistent spot, https://github.com/Azure/ARO-RP/blob/master/pkg/operator/const.go